.gitpod.yml

```
@@ -0,0 +1,31 @@
+tasks:
+  - init: |
+      # remove once latest maven works again or we switch to maven wrapper
+      sed -i 's/sdkman_auto_answer=false/sdkman_auto_answer=true/' $HOME/.sdkman/etc/config
+      sdk selfupdate
+      sdk update
+      sdk install maven 3.8.4
+      sdk use maven 3.8.4
+
+      brew install gh
+
+      mvn -am -pl war,bom -P quick-build install
+    command: |      
+      cd war
+      curl https://gist.githubusercontent.com/timja/fe73f0a0f98afe4d9356e7ce6752d95b/raw/98ad71080e91ed75824ec410ff905419e3788065/init.sh | bash
+      mvn jetty:run -Dhost=0.0.0.0
+    name: Run
+  - command: gp await-port 63342 && gp await-port 8080 && gp url 8080 && gp preview $(gp url 8080)/jenkins/
```

Hi, @timja!

The issue with "gp preview" not working in JetBrains IDEs has been fixed and deployed (https://github.com/gitpod-io/gitpod/issues/10118), so you can remove `gp await-port 63342` from from here:

```suggestion
  - command: gp await-port 8080 && gp url 8080 && gp preview $(gp url 8080)/jenkins/
```

It can also be removed from [jenkinsci/design-library-plugin's .gitpod.yml](https://github.com/jenkinsci/design-library-plugin/blob/4b1d2dd3da7ecbf61e4e3a466575a23dddad2adc/.gitpod.yml#L8).

---

thanks!